jac-uk/admin
# #2014 Fix Associated With Equality And Diversity Changes

## What's included?
As part of the work for https://github.com/jac-uk/admin/issues/1952 the social mobility questions can be either firstGenerationUniversity or parentsAttendedUniversity.
The Diversity view does not account for this change, it only displays firstGenerationUniversity. If this is not present in the data then an error is displayed in the console.

closes #2014 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Select an exercise which has a report
Click on the 'Reports' tab
The error will appear in the console

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context

<img width="1728" alt="Equality And Diversity Changes Bug" src="https://github.com/jac-uk/admin/assets/115651787/e613cc51-fced-475f-a80d-18f534e16d81">

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
